Source/API/termination_condition.h

```diff
@@ -8,8 +8,8 @@ class termination_condition {
 public:
     virtual ~termination_condition() = default;
 
-    virtual bool operator()(population* population) = 0;
-    virtual bool invoke(population* population) = 0;
+    virtual bool operator()(const population* const population) const = 0;
```

Comment:
it would be nice if the identifier had a different name then the type.

Reply:
Are you talking about the argument name? Wouldn't this be fixed in the type rename branch since casing would be different?

Reply:
This will be resolved when we change the casing of classes and structs.